cmd/crc/cmd/start.go

```diff
@@ -29,7 +29,7 @@ func init() {
 
 var startCmd = &cobra.Command{
 	Use:   "start",
-	Short: "start cluster",
+	Short: "Start cluster",
 	Long:  "Start cluster",
```

Comment:
long: `Start OpenShift 4.x cluster from the system bundle` or similar

Reply:
emphasize on 'the' as there is no 'an` (refering to a possible multiple) option

Reply:
I'd hide the bundle notion as much as possible from the ui, so not sure about the "from the system bundle" part, especially as this can be changed with -b.

Reply:
`Start the OpenShift cluster` works for me

Reply:
If we start using "the OpenShift cluster" instead of just cluster, we'd need to use it consistently across the board then:

  config      Modify crc configuration
  console     Access the OpenShift Web Console
  delete      Delete the OpenShift cluster
  help        Help about any command
  ip          Get IP address of the running OpenShift cluster
  oc-env      Add the 'oc' binary to PATH
  setup       Setup hypervisor
  start       Start the OpenShift cluster
  status      Display status of the OpenShift cluster
  stop        Stop the OpenShift cluster
  version     Print the version of crc

Reply:
Good point, and I think we should.